src/components/ValidatedInput/ValidSelectAsync.tsx

```diff
@@ -24,6 +26,8 @@ export interface ValidSelectAsyncProps<OptionType, Group, Additional, isMulti ex
   helperText?: React.ReactNode;
 }
 
+export type ValidSelectAsyncLoadOptions<OptionType> = LoadOptions<OptionType, unknown, unknown>;
```

Comment:
just curious where will we use this?

Reply:
We can use it to declare props in AP. for the example, in `GameServerSection`, I add it for `deploymentLoadOptions` props